static/app/components/externalIssues/abstractExternalIssueForm.tsx

```diff
@@ -348,17 +350,25 @@ export default class AbstractExternalIssueForm<
                     ...fields,
                     noOptionsMessage: () => 'No options. Type to search.',
                   }))
-                  .map(field => (
-                    <FieldFromConfig
-                      disabled={this.state.reloading}
-                      field={field}
-                      flexibleControlStateSize
-                      inline={false}
-                      key={`${field.name}-${field.default}-${field.required}`}
-                      stacked
-                      {...this.getFieldProps(field)}
-                    />
-                  ))}
+                  .map(field => {
+                    return (
+                      <React.Fragment
+                        key={`${field.name}-${field.default}-${field.required}`}
+                      >
+                        <FieldFromConfig
+                          disabled={this.state.reloading}
+                          field={field}
+                          flexibleControlStateSize
+                          inline={false}
+                          stacked
+                          {...this.getFieldProps(field)}
+                        />
+                        {errors.find(e => e === field.name) ? (
+                          <FieldErrorLabel>{`Could not fetch saved option for ${field.label}. Please reselect.`}</FieldErrorLabel>
+                        ) : null}
```

Comment:
It looks like we're taking in `errors` from props, but they aren't actual errors, just names of fields that have errors? 

What I worry about with this approach is someone extending AbstractExternalIssueForm and passing in something to the errors prop on this form and being surprised by the identical error message appearing on a bunch of fields when it might not actually be helpful/accurate.

I don't know how we do it elsewhere, but I'd prefer something like a dictionary in the shape of `{fieldName: errorMessage}` and with a helpful TS type so that it can be used in a bit more of a generic way.